subsys/net/l2/ppp/lcp.c

```diff
@@ -203,6 +203,100 @@ static void lcp_finished(struct ppp_fsm *fsm)
 	ppp_link_terminated(ctx);
 }
 
+#if defined(CONFIG_PPP_DIALUP)
+
+#define MRU_OPTION_LEN 4
+#define MRU_DATA_LEN 2
+
+static int lcp_add_mru(struct ppp_context *ctx, struct net_pkt *pkt)
+{
+	uint8_t data[MRU_DATA_LEN];
+	
+	sys_put_be16(ctx->lcp.my_options.mru, data);
+
+	net_pkt_write_u8(pkt, MRU_OPTION_LEN);
+	return net_pkt_write(pkt, data, sizeof(data));
```

Comment:
You can simply write this as ` net_pkt_write_be16(pkt, ctx->lcp.my_options.mru);`